### PR TITLE
PRIME-2171 New version of Pharm tech TOA

### PR DESCRIPTION
**Pull request by AMiKaH**
[PRIME-2171](https://bcgovmoh.atlassian.net/browse/PRIME-2171)

### CODE REVIEW

File: prime-dotnet-webapi/Configuration/Database/Resources/pharmacy-tech-ru-no-obo-toa-v2.html [hunk 1]
```diff
@@ -0,0 +1,651 @@
...
+          with section 10.
+        </p>
+
+      </li>
+    </ol>
+
+  </li>
+</ol>
+<p>
+  Version date: January 28, 2022
```
Comment (itchison) [inline prime-dotnet-webapi/Configuration/Database/Resources/pharmacy-tech-ru-no-obo-toa-v2.html:650]: Is this date on the other TOAs?

Comment (AMiKaH) [reply prime-dotnet-webapi/Configuration/Database/Resources/pharmacy-tech-ru-no-obo-toa-v2.html:650]: Yes, it's at the bottom of the word file:
image

Comment (itchison) [reply prime-dotnet-webapi/Configuration/Database/Resources/pharmacy-tech-ru-no-obo-toa-v2.html:650]: Yes please remove it. @AMiKaH

**Comment by AMiKaH** [comment]
I recall that we're replacing the old ToA and not adding a new version so I replaced the text in place. Awaiting confirmation.

File: prime-dotnet-webapi/Configuration/Database/Resources/pharmacy-tech-ru-no-obo-toa-v1.html [hunk 2]
```diff
@@ -388,7 +388,7 @@ <h1>PHARMANET TERMS OF ACCESS FOR PHARMACY TECHNICIANS</h1>
   <li>
 
     <p class="bold underline">
-      NOTICE OF NON COMPLIANCE AND DUTY TO INVESTIGATE
+      NOTICE OF NON-COMPLIANCE
```
[the remark this answers is not in the document]
Comment (AMiKaH) [reply prime-dotnet-webapi/Configuration/Database/Resources/pharmacy-tech-ru-no-obo-toa-v1.html:391]: From memory, I remember it was a replacement to the existing one as opposed to a newer version also from what the ticket mentions. I am waiting for confirmation from Nick.

**Comment by AMiKaH** [comment]
Confirmed, new version. Updated new TOA to v2 with effective date of March 16, 2022

**Review by johnnyl6 — COMMENTED** [review]
LGTM

File: prime-dotnet-webapi/Migrations/20220316172231_AddNewPharmacyTechnicianToa.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
...e, if any; or\r\n          </li>\r\n          <li>\r\n            by publishing notice of any such amendment in the PharmaCare\r\n            Newsletter, in which case the notice will specify the effective date of the\r\n            amendment, which date will be at least 30 (thirty) days after the date that\r\n            the PharmaCare Newsletter containing the notice is first published.\r\n          </li>\r\n        </ol>\r\n\r\n        <p>\r\n          If you use PharmaNet after the effective date of an amendment described in (i) or\r\n          (ii) above, you will be deemed to have accepted the corresponding amendment,\r\n          and this Agreement will be deemed to have been so amended as of the effective\r\n          date. If you do not agree with any amendment for which notice has been provided\r\n          by the Province in accordance with (i) or (ii) above, you must promptly (and in\r\n          any event before the effective date) cease all access or use of PharmaNet by yourself and take the steps\r\n          necessary to terminate this Agreement in accordance\r\n          with section 10.\r\n        </p>\r\n\r\n      </li>\r\n    </ol>\r\n\r\n  </li>\r\n</ol>\r\n");
```
Comment (neophyte57) [inline prime-dotnet-webapi/Migrations/20220316172231_AddNewPharmacyTechnicianToa.cs:15]: I am wondering why in the generated migration there is an update to the previous Pharm Tech TOA?

Comment (AMiKaH) [reply prime-dotnet-webapi/Migrations/20220316172231_AddNewPharmacyTechnicianToa.cs:15]: Good point, looking into it

Comment (neophyte57) [reply prime-dotnet-webapi/Migrations/20220316172231_AddNewPharmacyTechnicianToa.cs:15]: You might have to do these steps to rollback and re-generate your migration:
1. `dotnet ef database update <migration prior to your changes> --verbose` (https://docs.microsoft.com/en-us/ef/core/cli/dotnet#dotnet-ef-database-update)
2. Overwrite `ApiDbContextModelSnapshot.cs` with the content of version prior to your changes
3. Delete `prime-dotnet-webapi/Migrations/20220316172231_AddNewPharmacyTechnicianToa.cs`
4. Run `dotnet ef migrations add <name of your migration>`

Then you have to commit the deletion of `prime-dotnet-webapi/Migrations/20220316172231_AddNewPharmacyTechnicianToa.cs` so it doesn't show up in your PR.

Comment (AMiKaH) [reply prime-dotnet-webapi/Migrations/20220316172231_AddNewPharmacyTechnicianToa.cs]: Fixed

**Review by neophyte57 — APPROVED** [review]
👍

**Review by hogmann — APPROVED** [review]
LGTM

**Pull request merged**